BlueprintUICommonControls/Sources/TransitionContainer.swift

```diff
@@ -1,31 +0,0 @@
-import BlueprintUI
-import UIKit
-
-
-/// Wraps a content element and adds transitions when the element appears,
-/// disappears, or changes layout.
-public struct TransitionContainer: Element {
```

Comment:
This is now `Transition`.

Reply:
I removed `Container` because basically every type that acts as a wrapper is a container, but no others have `container` in their name – eg `Inset` isn't `InsetContainer`, etc. I like `Transitioning`, I think

Reply:
Or maybe `OnTransition(onAppear:onDisappear:)`?